spec/services/container_dashboard_service_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "spec_helper"
+
+describe ContainerDashboardService do
+  before(:each) do
+    ManageIQ::Providers::ContainerManager.delete_all
+  end
+
+  context "providers" do
+    it "returns all providers correctly" do
+      Tempfile.create(['config', 'yml']) do |f|
```

Comment:
why is there a need to create a file during test? typically this won't run properly on all test envs. 
iirc @simon3z also commented on this.